annotations/templates/annotations/project_list.html

```diff
@@ -5,6 +5,38 @@
 <script src="{% static "annotations/js/spin.min.js" %}"></script>
 <script src="{% static "annotations/js/jquery.spin.js" %}"></script>
 <script src="{% static "annotations/js/cytoscape.min.js" %}"></script>
+
+<script>
+$(document).ready(function() {
+    // Handle clicks on project links for text import
+    $('.project-import-link').click(function(e) {
+        e.preventDefault();
+        var url = $(this).attr('href');
+        
+        // loading spinner
+        $(this).spin();
+        
+        // Make AJAX call to import text
+        $.ajax({
```

Comment:
Does this really have to be an ajax call? it seems like the view could redirect whereever it needs to go (that would also solve the issue I have below).